documentation/dashboard/trials.md

```diff
@@ -297,7 +301,7 @@ the solarforecastarbiter-core Python package can be installed from the
 or via pip with the command
 
 ```
-pip install git+https://github.com/solararbiter/solarforecastarbiter-core.git
+pip install solarforecastarbiter
```

Comment:
needs `--pre`?

Reply:
didnt' for me. can you test too?

Reply:
indeed. https://www.python.org/dev/peps/pep-0440/#handling-of-pre-releases

> Pre-releases of any kind, including developmental releases, are implicitly excluded from all version specifiers, unless they are already present on the system, explicitly requested by the user, or if the only available version that satisfies the version specifier is a pre-release.